core/pta/tests/invoke.c

```diff
@@ -277,8 +277,8 @@ static TEE_Result test_inject_sdp(uint32_t type, TEE_Param p[TEE_NUM_PARAMS])
 		return TEE_SUCCESS;
 
 	/* Check that core can p2v and v2p over memory reference arguments */
-	if (test_v2p2v(src) || test_v2p2v(src + sz - 1) ||
-	    test_v2p2v(dst) || test_v2p2v(dst + sz - 1))
+	if (test_v2p2v(src, sz) || test_v2p2v(src + sz - 1, 0) ||
+	    test_v2p2v(dst, sz) || test_v2p2v(dst + sz - 1, 0))
```

Comment:
passing `0` size is strange, and actually, the 1st test already does both jobs.
What about `if (test_v2p2v(src, sz) || test_v2p2v(dst, sz))` ?

Reply:
Looks like here must be passed 1 instead of 0 - to check last byte in range

Reply:
Old `test_v2p(va)` only checked 1 single memory cell address conversion, so we used check both start and end addresses of the buffer.

Since now `test_v2p2v(va, sz)` checks base address + buffer consistency in virtual space, i don't think it is necessary to test the address of the buffer's last memory cell, hence only 1 call to `test_v2p()`.

Reply:
You`re right, first check covers full range. Fixed this.